secrets.js

```diff
@@ -0,0 +1,5 @@
+const infuraAccessToken = 'DO NOT COMMIT REAL VALUES!';
```

Comment:
it seems that this might have been accidentally committed?

Reply:
no, this is just a notice/warning just in case. 

I found and run this command:
`git update-index --skip-worktree secrets.js`
so it prevents tracking changes for the file - so file will not be listed as modified.

Reply:
maybe we add 
```
npm install --save dotenv
```

Reply:
maybe I will add this info to the file itself, for future reference

Reply:
I dont think we need the secret.js file instead list the required environment variables.

Reply:
we can even remove the comments by adding a conditional to check what env is provided.

Reply:
about `dotenv` - what do we gain? we still will have additional file that should not be committed with real values. and in order not to create the file we need to use this git command to stop tracking the chnages.

Reply:
> we can even remove the comments by adding a conditional to check what env is provided.

you can check env in `truffle.js`?